### DISCUSSION TITLE
Monitoring connection pool

### DISCUSSION THREAD

**Discussion by thpoiani** [sequelize/sequelize#14236] [Help & Questions]
👋 Hello community, hope you are doing great.
Do we have any way to **monitor the connection pool**?
Would like to see connections being added and released, maybe even using Prometheus.
Should I implement it manually through `afterConnect` and `afterDisconnect` hooks?

**Comment by fzn0x** [comment]
I think it's related to https://github.com/sequelize/sequelize/issues/13660

Isn't it?

**Reply by thpoiani** [reply]
Seems related ✔️
